src/main/java/homework_5/custom_regex_matcher/CustomRegexMatcher.java

```diff
@@ -0,0 +1,27 @@
+package homework_5.custom_regex_matcher;
+
+import java.util.Scanner;
+
+public class CustomRegexMatcher {
+
+    private CustomRegexMatcher() { }
+
+    private static final String PHONE_NUMBER_FORMAT = "(\\+\\d)\\s\\(\\d{3}\\)\\s(\\d{3})-(\\d{4})";
```

Comment:
Opt: You could add some variability to regex (like, accept phone numbers with different country codes and etc)